app/js/components/topics/topicRuleList/topicRuleList.directive.js

```diff
@@ -21,7 +21,7 @@ function topicRuleListCtrl ($filter,
     $scope.listTypes = ListTypeService.types();
     $scope.getListType = ListTypeService.getType;
     $scope.sorter = new Utils.Sorter({
-        predicate:['acked', 'hitCount === 0', '-severityNum', '-hitCount'],
```

Comment:
@Kinlaw the idea here was that for the best user experience, the rules in a topic should be sorted in a certain way. The rules are: acked rules and rules with no hits go last, rules with higher severity and more hits preferred. This was expressed in the default predicate.

The sorter util does not support indication of such complicated predicate. That means that no table header column arrow will be highlighted by default. Still, you can click on column headers to start sorting the table yourself by a single predicate, which kills the default complicated predicate. The sorting indication should work as expected in this case.

This is not good because:
1) There's no way for a user to re-establish the complicated predicate that comes by default
2) By default there is no indication on what columns are used for ordering

Still, I think the former predicate it the best way to sort the topic rules by default. If the two aforementioned points are a problem we should IMHO rather work on addressing those (by better indication + making it possible for users to sort by multiple columns) rather than making the usability worse by changing the default predicate.

WDYT?

Reply:
That's a good point. I didn't think about the consequences of removing the other sort criteria. I'll switch it back and proceed with the PR as is so that we can get the sort indication working on the other tables. I'll fix this with the implementation of multi-column sorting.